packages/flutter/lib/src/painting/rounded_rectangle_border.dart

```diff
@@ -126,12 +126,17 @@ class RoundedRectangleBorder extends OutlinedBorder {
       case BorderStyle.none:
         break;
       case BorderStyle.solid:
-        final Paint paint = Paint()
-          ..color = side.color;
-        final RRect borderRect = borderRadius.resolve(textDirection).toRRect(rect);
-        final RRect inner = borderRect.deflate(side.strokeInset);
-        final RRect outer = borderRect.inflate(side.strokeOutset);
-        canvas.drawDRRect(outer, inner, paint);
+        if (side.width == 0.0) {
+          canvas.drawRRect(borderRadius.resolve(textDirection).toRRect(rect), side.toPaint());
+        } else {
+          final Paint paint = Paint()
+            ..color = side.color;
+          final RRect borderRect = borderRadius.resolve(textDirection).toRRect(rect);
+          final RRect inner = borderRect.deflate(side.strokeInset);
+          final RRect outer = borderRect.inflate(side.strokeOutset);
+          canvas.drawDRRect(outer, inner, paint);
+        }
+        break;
```

Comment:
Is `canvas.drawRRect` better than `break;`? I'm curious if there was a difference.

Your `break;` in 139 is 'unecessary' (if you add, you might want to add in 130, after drawRRect, and on the other borders. But `break` = nothing so it doesn't matter.

Reply:
I realize that it doesn't do anything, but it provides consistency with the other case statement, and if a new case is added, the person adding it won't need to add a `break`.

Reply:
Oh, sure. Feel free to add to the other borders if you feel it is relevant, I think only the Continous or Beveled have it. The other ones don't.